lib/ansible/plugins/callback/sentry.py

```diff
@@ -0,0 +1,108 @@
+# (C) 2018 Filippo Giunchedi <filippo@esaurito.net>
+# (C) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+    callback: sentry
+    type: notification
+    short_description: Sends errors to Sentry
+    description:
+      - Send Ansible errors and exceptions to Sentry
+    version_added: "2.7"
+    requirements:
+      - whitelisting in configuration
+      - sentry-sdk (python library)
+"""
+
+from datetime import datetime
+from ansible.plugins.callback import CallbackBase
+
+
+try:
+    import sentry_sdk as sentry
+
+    HAS_SENTRY = True
+except ImportError:
+    HAS_SENTRY = False
+
+
+class CallbackModule(CallbackBase):
+    """
+    Requires:
+        sentry-sdk
+
+    Environment:
+        SENTRY_DSN (required): Sentry DSN
+    """
+
+    CALLBACK_VERSION = 0.1
```

Comment:
this is not the version of the callback itself, but the API, 2.0 would be the correct number for this usage

Reply:
Whoops! Fixed.